script/deploy/mainnet/M2_Mainnet_Upgrade.s.sol

```diff
@@ -0,0 +1,308 @@
+                delegationManagerImplementation
+            )
+        );
+
+        txs[1] = Tx(
+            address(eigenLayerProxyAdmin),
+            0,
+            abi.encodeWithSelector(
+                ProxyAdmin.upgrade.selector, 
+                TransparentUpgradeableProxy(payable(address(slasher))), 
+                slasherImplementation
+            )
+        );
+
+        txs[2] = Tx(
+            address(eigenLayerProxyAdmin),
+            0,
+            abi.encodeWithSelector(
+                ProxyAdmin.upgrade.selector, 
+                TransparentUpgradeableProxy(payable(address(strategyManager))), 
+                strategyManagerImplementation
+            )
+        );
+
+        txs[3] = Tx(
+            address(eigenLayerProxyAdmin),
+            0,
+            abi.encodeWithSelector(
+                ProxyAdmin.upgrade.selector, 
+                TransparentUpgradeableProxy(payable(address(delayedWithdrawalRouter))), 
+                delayedWithdrawalRouterImplementation
+            )
+        );
+
+        txs[4] = Tx(
+            address(eigenLayerProxyAdmin),
+            0,
+            abi.encodeWithSelector(
+                ProxyAdmin.upgrade.selector, 
+                TransparentUpgradeableProxy(payable(address(eigenPodManager))), 
+                eigenPodManagerImplementation
+            )
+        );
+
+        txs[5] = Tx(
+            address(eigenPodBeacon),
+            0,
+            abi.encodeWithSelector(
+                UpgradeableBeacon.upgradeTo.selector, 
+                eigenPodImplementation
+            )
+        );
+
+        // unpause everything on DelegationManager
+        txs[6] = Tx(
+            address(delegationManager), 
+            0, // value
+            abi.encodeWithSelector(Pausable.unpause.selector, 0)
+        );
+
+        // set the min withdrawal delay blocks on the DelegationManager
+        txs[7] = Tx(
+            address(delegationManager), 
+            0, // value
+            abi.encodeWithSelector(DelegationManager.setMinWithdrawalDelayBlocks.selector, DELEGATION_MANAGER_MIN_WITHDRAWAL_DELAY_BLOCKS)
+        );
+
+        // unpause everything on EigenPodManager
+        txs[8] = Tx(
+            address(eigenPodManager), 
+            0, // value
+            abi.encodeWithSelector(Pausable.unpause.selector, 0)
+        );
+
+        // set beacon chain oracle on EigenPodManager
+        txs[9] = Tx(
+            address(eigenPodManager), 
+            0, // value
+            abi.encodeWithSelector(EigenPodManager.updateBeaconChainOracle.selector, IBeaconChainOracle(beaconChainOracle))
+        );
+
+        // set Deneb fork timestamp on EigenPodManager
+        txs[10] = Tx(
+            address(eigenPodManager), 
+            0, // value
+            abi.encodeWithSelector(EigenPodManager.setDenebForkTimestamp.selector, EIGENPOD_MANAGER_DENEB_FORK_TIMESTAMP)
+        );
```

Comment:
Same request here - configure, then unpause.

Reply:
adjusted the ordering in https://github.com/Layr-Labs/eigenlayer-contracts/pull/480/commits/9b7a0c33c94c20f2e5bb8abfacf115e15bf24426